### ISSUE TITLE
The 'translators:' comment breaks assignment alignment rule

### ISSUE THREAD

**Issue by msaggiorato** [WordPress-Coding-Standards/WordPress-Coding-Standards#1312]
I've been running this and the example below is breaking

```php
$this->id                 = 'asd';
$this->method_title       = __( 'asd', 'test-domain' );
/* translators: %s: URL to X */
$this->method_description = sprintf( __( 'This triggers translators because %s', 'test-domain' ), 'https://me.sumup.com' );
$this->has_fields         = true;
$this->supports           = array(
```

I'm currently using the latest PHPCS and the latest sniffs aswell.

**Comment by jrfnl** [comment]
What do you mean by "breaking" ?
What is the error you are receiving ?

**Comment by GaryJones** [comment]
Your translator comment lists two placeholders, but your string pattern only has one.

For single placeholders, numbered placeholders aren't needed. You could change `%1$s` to just `%s`.

If you did need multiple placeholders, then see https://developer.wordpress.org/plugins/internationalization/how-to-internationalize-your-plugin/#descriptions for the format (uses numbers followed by colons, rather than numbered placholders followed by `=`._

**Comment by GaryJones** [comment]
> I was under the impresion that the alignment rules shouldn't be affected by the presence of this type of comment.

The presence of the comment makes the lines containing `=` symbols non-consecutive, just the same as a clear linespace would.

You could always just re-order the lines, so that the translator comment is top.

**Comment by msaggiorato** [comment]
Thanks Gary ;)
